sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2019,8 +2019,7 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setBigQueryServices(testServices).build();
     }
 
-    @VisibleForTesting
-    Write<T> withMaxFilesPerBundle(int maxFilesPerBundle) {
+    public Write<T> withMaxFilesPerBundle(int maxFilesPerBundle) {
```

Comment:
Can you please document the usage of this function ?

Reply:
Done.